frontend/public/manifest.json

```diff
@@ -0,0 +1,25 @@
+{
+  "short_name": "GreedyBear",
+  "name": "GreedyBear",
+  "icons": [
+    {
+      "src": "favicon.ico",
```

Comment:
we need to substitute this default favicon with the one of Honeynet that you may find in `gui/static/greedybear.png`